### PR TITLE
Manually merge 2.11.x into 3.0.x

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: lib/Doctrine/DBAL/Portability/Connection.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<?php
```
Comment (morozov) [inline lib/Doctrine/DBAL/Portability/Connection.php:1]: This file should be moved to `src/Portability/Connection.php`. Apparently, depending on the number of changes, Git doesn't always understand the rename.

**Review by morozov — CHANGES_REQUESTED** [review]

File: lib/Doctrine/DBAL/Portability/Connection.php (hunk 1 above)
Comment (greg0ire) [reply lib/Doctrine/DBAL/Portability/Connection.php:1]: Woops! Yes, I saw it managed to do it for some other files.

Comment (greg0ire) [reply lib/Doctrine/DBAL/Portability/Connection.php]: Oh and actually I applied the changes to the destination file, bug forgot to remove the source file :facepalm:

Comment (morozov) [reply lib/Doctrine/DBAL/Portability/Connection.php]: In this case, I usually accept the files in `lib/` and move them manually to `src/`. Then I have the actual diff and can fine-tune it. Shouldn't be happening that often though.

**Review by morozov — APPROVED** [review]

**Comment by morozov** [comment]
Thanks, @greg0ire!

**Pull request merged**